### ISSUE TITLE
`/var/lib/warewulf/chroots/overlay` directory should be somewhere else.

### ISSUE THREAD

**Issue by griznog** [warewulf/warewulf#1180]
### Steps to reproduce

```
[root@micropterus chroots]# ls
rocky-9  rocky-9-minimal
[root@micropterus ~]# wwctl container shell rocky-9
[rocky-9] Warewulf> exit
exit
+ LANG=C
+ LC_CTYPE=C
+ export LANG LC_CTYPE
+ dnf clean all
0 files removed
Rebuilding container...
Created image for VNFS container rocky-9: /var/lib/warewulf/provision/container/rocky-9.img
Compressed image for VNFS container rocky-9: /var/lib/warewulf/provision/container/rocky-9.img.gz
[root@micropterus chroots]# ls 
overlays  rocky-9  rocky-9-minimal
[root@micropterus chroots]# tree overlays/
overlays/
├── lower
├── nodeoverlay
└── work

3 directories, 0 files
```

Not sure what this `overlay` directory does or what triggers it to be created, but it breaks the assumption that everything in `/var/lib/warewulf/chroots` will be a node image `chroot`.

**Comment by anderbubble** [comment]
@mslacken can you look into this? I'd like to try to get a fix for this in v4.5.1 if possible.

**Comment by mslacken** [comment]
Yes this are workdirs for the chroot process, especially if you the `--render  $NODENAME` option, I will make a PR which removes them.
